Incorrect status of "Big object parts are removed after deletion" test

Object parts check can result in:

> AssertionError: Regex pattern did not match.
 Regex: 'code = 2052.*message = object already removed'
 Input: "Command: ./neofs-cli --config /home/runner/work/neofs-node/neofs-node/neofs-testcases/wallet_config.yml object get --rpc-endpoint 'localhost:57329' --wallet '/home/runner/work/neofs-node/neofs-node/neofs-testcases/test-run-2025-12-18-22-12-55-369703/wallet-9e96533b-6b79-4579-b27c-588a1907d47f.json' --cid 'J4zVqYeSGE8zMDNdUjasi1ZTMYKnPSW3YM4njCYihiqo' --oid '61VkVXiJXL3zz9wmN6B8oR4yoazathkdYgLvUWJizL8G' --file '/home/runner/work/neofs-node/neofs-node/neofs-testcases/test-run-2025-12-18-22-12-55-369703/TestObjectsDir/9be9e980-a556-458e-a931-b89ae1c9c494' --no-progress\nError:\nreturn code: 1\noutput: Error: rpc error: read header: status: code = 2049 message = object not found\n"

This is not correct for the test. Object parts can be deleted by GC and they have weak links to the deleted parent, so 2049 is _also_ acceptable here, the main point of the test is that these objects are gone and they are.

--- a/pytest_tests/tests/object/test_object_api.py
+++ b/pytest_tests/tests/object/test_object_api.py
@@ -948,7 +948,7 @@ class TestObjectApi(TestNeofsBase):
 
         with allure.step("Try to get object parts"):
             for part in parts:
-                with pytest.raises(Exception, match=OBJECT_ALREADY_REMOVED):
+                with pytest.raises(Exception, match=f"({OBJECT_ALREADY_REMOVED}|{OBJECT_NOT_FOUND})"):
                     get_object(
                         default_wallet.path,
                         container,
@@ -997,7 +997,7 @@ class TestObjectApi(TestNeofsBase):
 
         with allure.step("Try to get object parts"):
             for part in parts:
-                with pytest.raises(Exception, match=OBJECT_NOT_FOUND):
+                with pytest.raises(Exception, match=f"({OBJECT_ALREADY_REMOVED}|{OBJECT_NOT_FOUND})"):
                     get_object(
                         default_wallet.path,
                         container,

